modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/model/StyledTextModel.java

```diff
@@ -1012,23 +1012,26 @@ private void checkWritable() {
 
     /**
      * Specifies the line ending characters.
-     * A {@code null} value results in the platform line ending as reported by
-     * the {@code line.separator} system property.
      *
      * @return the line ending value
-     * @defaultValue null
+     * @defaultValue {@link LineEnding#SYSTEM}
      * @since 26
      */
     public final LineEnding getLineEnding() {
+        if (lineEnding == null) {
+            lineEnding = LineEnding.SYSTEM;
+        }
```

Comment:
This isn't needed, right? `lineEnding` can never be null as long as you initialize it to `LineEnding.SYSTEM `, since you check and throw on an attempt to set it to null.